smriprep/workflows/anatomical.py

```diff
@@ -231,15 +241,18 @@ def init_anat_preproc_wf(skull_strip_template, fs_spaces, template, debug,
 
     # 3. Skull-stripping
     # Bias field correction is handled in skull strip workflows.
-    skullstrip_ants_wf = init_skullstrip_ants_wf(name='skullstrip_ants_wf',
-                                                 skull_strip_template=skull_strip_template,
-                                                 debug=debug,
-                                                 omp_nthreads=omp_nthreads)
+    bran_extraction_wf = init_brain_extraction_wf(
```

Comment:
```suggestion
    brain_extraction_wf = init_brain_extraction_wf(
```

Reply:
OMG never push a commit while you hear your kid crying :(